dev.sh

```diff
@@ -327,7 +327,7 @@ systemd_reload() {
     dst=$2
 
     echo >&2 "staging ${SUBSTRATEOS_OVERLAY_IMAGE_DIR}$src"
-    (cd ${SUBSTRATEOS_OVERLAY_IMAGE_DIR}$src; find .)
+    (sudo cd ${SUBSTRATEOS_OVERLAY_IMAGE_DIR}$src; find .)
```

Comment:
this was to fix a permission error on this line

Reply:
does sudo cd work? that seems like it would have no effect in a shell

Reply:
Hmm, yeah it probably just has no effect, which in this case simply bypassed the failing command. It seems like after re-imaging the machine I may not need that change anymore.

Reply:
ok, let's drop this change then